test/SpecHelper.hs

```diff
@@ -81,6 +81,8 @@ _baseCfg =  -- Connection Settings
             []
             -- No root spec override
             Nothing
+            -- Raw output media types
+            ["text/html"]
```

Comment:
What we should do is to have another config(smth like raw-media-types.config) that includes this setting, rest of the tests should run with raw-media-types of `[]`.

Then you should move this test https://github.com/PostgREST/postgrest/blob/master/test/Feature/RpcSpec.hs#L467-L482 to its own Spec and have this spec use the previous config.

Reply:
@steve-chavez yep, I am working on it at the moment. I am hoping to push this before end of day.